### ISSUE TITLE
Alt text on infographic promo image

### ISSUE THREAD

**Issue by mworrest** [NCIOCPL/cgov-digital-platform#2704]
## As a user, if I am using a screen reader and an infographic is featured in a feature card, I want my screen reader to read me alt text specific to the promotional image, so I fully understand the content I’m being presented. 

> **ESTIMATE** 2

### Acceptance Criteria

```gherkin
Scenario: Infographic in a feature card pulls correct alt text
  Given The user is using a screen reader
    And The infographic is featured in a card
    And The 4:3 or 16:9 promotional image associated with the infographic displays
  When  The user accesses the alt text
  Then  The alt text assigned to the infographic will be read
```
```gherkin
Scenario: User wants to view the alt text of an Infographic in a feature card (using view source)
  Given The user is viewing a page with an infographic
    And The infographic is featured in a card
    And The 4:3 or 16:9 promotional image associated with the infographic displays
  When  The user views the page source
  Then  the source code has alt text for the infographic displayed
    And the alt text is the one associated with the infographic promo image
  Then  The alt text assigned to the infographic will be read 
```

### Resources:
* Feature: https://tracker.nci.nih.gov/browse/CGOV-11460
* Mockups: https://publicis.invisionapp.com/share/CJW8VVAV87U
* Web Page: https://www.cancer.gov/about-nci

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- #2705 - this ticket has been completed for feature cards and has been updated with that; since we're still working on lists, the ticket is still open.

### Tasks
- [ ]  Add the infographic alt text to : docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/cards/media--cgov-infographic--multimedia-card.html.twig -AC

### Technical Notes
N/A

**Comment by meglake** [comment]
This story is on hold as it is dependent on 2705 which is also on hold.

@claireblaustein  as time permits, please update criteria to be in line with new alt text criteria you are creating in 2703.

**Comment by meglake** [comment]
Moving to redesign - design ticket project for consideration during redesign work.

**Comment by sarinapadilla** [comment]
Closing this ticket, as it has been implemented with #3637.
